sshh/main.py

```diff
@@ -110,10 +111,32 @@ def cmd_agent(request):
         sshenv['PS1'] = f"[{request.group}]{sshenv['PS1']}"
         logger.info('ssh-agent PID=%s session "%s" has been started. To close this session, exit shell.',
                     sshenv['SSH_AGENT_PID'], request.group)
-        subprocess.run (sshenv['SHELL'], env=sshenv)
+        shell_command = sshenv['SHELL']
+        bashrc_location = Path(sshenv.get('HOME', ''), '.bashrc')
+        if Path(shell_command).name.lower() == 'bash' and bashrc_location.is_file():
+            # set PS1 after bash execution if shell program is bash and if bashrc is found
+            temp_rc = mkstemp()
+            with open(temp_rc[1], 'w') as rc_file:
+                # Write temp rc file that sets PS1 after loading bashrc
+                rc_file.writelines([f'. {bashrc_location}\n',
+                                    f'PS1="{sshenv["PS1"]}"'])
+            # Set command to override rcfile with tempfile
+            shell_command = [shell_command,
+                             '--rcfile',
+                             temp_rc[1]]
+        else:
+            # For saefly cleaning up temp files
+            temp_rc = None
+        subprocess.run(shell_command, env=sshenv)
     finally:
         # kill agent
         subprocess.run(['ssh-agent', '-k'], env=sshenv, stdout=subprocess.DEVNULL)
+        if temp_rc:
+            # Close temp files
+            # temp_rc[0] = file handler id
+            # temp_rc[1] = temp file path
+            os.close(temp_rc[0])
+            Path(temp_rc[1]).unlink()
```

Comment:
Maybe close and unlink is unnecessary since sshh is unlikely to persistently run or open extra files?

But it's probably safer to do this just in case.